localisation/00_wc_sad.csv

```diff
@@ -453,9 +453,6 @@ raise_host_of_the_wilds_power_desc;Enables decision in the 
 EVTDESC_WCCSO.10100;Ancients are powerful beings that have supported the Cenarion Circle for several millennia. As a member, you can ask one of these ancients to grant you their blessing.;;Urtume sind m�chtige Wesen, die den Zirkel des Zaren f�r Tausende von Jahren unterst�tzt haben. Als Mitglied k�nnt Ihr einen von ihnen bitten, Euch ihren Segen zu gew�hren.;;;;;;;;;;;x
 EVTOPTA_WCCSO.10100;I will ask an ancient of war.;;Ich werde ein Urtum des Krieges fragen.;;;;;;;;;;;x
 EVTOPTB_WCCSO.10100;I will ask an ancient of lore.;;Ich werde ein Urtum der Weisheit fragen.;;;;;;;;;;;x
-EVTDESC_WCOME.100;After the taking of city your troops start sacrificing its population to the Lords of Twilight. The streets are flooded with blood and screams of innocents blend into a symphony of terror. The madness has come to this land.;;Nach der Eroberung der Stadt beginnen Eure Truppen, ihre Bev�lkerung den Herren des Zwielichts zu opfern. Die Stra�en sind blut�berflutet und die Schreie von Unschuldigen verschmelzen zu einer Symphonie des Terrors. Der Wahnsinn ist in dieses Land gekommen.;;;;;;;;;;;x
-EVTOPTA_WCOME.100;Glory to the Gods Below.;;Ehre sei den G�ttern in der Tiefe.;;;;;;;;;;;x
-EVTOPTB_WCOME.100;Hide this event.;;Dieses Ereignis ausblenden.;;;;;;;;;;;x
```

Comment:
Please, don't remove thes lines. At least leave the lines with `LINE_REMOVED_DONT_TOUCH` in the first column.

Reply:
Done